azurerm/provider_test.go

```diff
@@ -12,6 +12,9 @@ import (
 	"github.com/hashicorp/terraform/terraform"
 )
 
+// ParaTestEnvVar is used to enable parallelism in testing.
+const ParaTestEnvVar = "TF_PARA"
```

Comment:
I was struggling the naming principles here:
1. TF_PARALLEL: clear but a bit longer while using it the command line.
2. TF_PARA: short to type in command line, but as you said, not quite clear, might confuse people.

Then I took a look at TF_ACC, I decided to use #2 to keep just one naming principle in the same source file.

@jeffreyCline , @JunyiYi what do you think ? I am open to use either of them.

Reply:
https://www.allacronyms.com/parallel/abbreviated

We decided to use "TF_PARL" instead according to above link.

Reply:
@metacpp whilst I understand that - we tend to be more descriptive in the environment variables we use - such as:

- `TF_WORKSPACE`
- `TF_IN_AUTOMATION`
- `TF_SKIP_PROVIDER_VERIFY`
- `TF_PLUGIN_CACHE_DIR`

As such I think this is best as `TF_PARALLEL` for consistency with the rest of Terraform - given this is an opt-in setting; when setting this as an environment variable this can be set once and forgotten (like the Provider environment variables are - as such, I don't think adding a couple of extra characters to be readable is a bad thing).